.github/workflows/build_pr_documentation.yml

```diff
@@ -9,6 +9,7 @@ concurrency:
 
 jobs:
   build:
+    if: github.repository == 'TimDettmers/bitsandbytes'
```

Comment:
can you elaborate on this change? Would this affect PRs from forks?

Reply:
Yes. I was running exploratory versions of this PR in my own fork, and the docs workflow wouldn't be necessary there.

Reply:
ok makes sense